.trampolinerc

```diff
@@ -48,3 +48,14 @@ fi
 
 # The build will show some commands and docker build logs.
 TRAMPOLINE_VERBOSE="true"
+
+public_image="gcr.io/cloud-devrel-public-resources/docker-ci-helper/python"
+function trampoline_upload_hook {
```

Comment:
If we plan to add more function hooks and repeat this pattern, let's come up with a consistent naming scheme to generalize it. (maybe something like `trampoline_<before|after>_<phase>`)

It can decouple the hook name from what's being invoked at that moment.

Reply:
Thanks, will do